package.json

```diff
@@ -28,33 +28,33 @@
   ],
   "author": "Vojta Jina <vojta.jina@gmail.com>",
   "dependencies": {
-    "jasmine-core": "^3.6.0"
+    "jasmine-core": "^4.1.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "^12.1.4",
-    "@commitlint/config-angular": "^12.1.4",
-    "@semantic-release/changelog": "^5.0.1",
-    "@semantic-release/git": "^9.0.1",
-    "@semantic-release/github": "^7.2.3",
-    "@semantic-release/npm": "^7.1.3",
-    "eslint": "^7.32.0",
+    "@commitlint/cli": "^16.2.3",
+    "@commitlint/config-angular": "^16.2.3",
+    "@semantic-release/changelog": "^6.0.1",
+    "@semantic-release/git": "^10.0.1",
+    "@semantic-release/github": "^8.0.4",
+    "@semantic-release/npm": "^9.0.1",
+    "eslint": "^8.13.0",
     "eslint-config-standard": "^14.1.1",
-    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.1",
-    "grunt": "^1.4.1",
-    "husky": "^4.2.5",
-    "jasmine": "^3.6.1",
-    "karma": "^4.4.1",
-    "karma-firefox-launcher": "^1.3.0",
-    "semantic-release": "^17.4.7"
+    "eslint-plugin-promise": "^6.0.0",
+    "eslint-plugin-standard": "^4.1.0",
+    "grunt": "^1.5.1",
+    "husky": "^4.3.8",
+    "jasmine": "^4.1.0",
+    "karma": "^6.3.17",
+    "karma-firefox-launcher": "^2.1.2",
+    "semantic-release": "^19.0.2"
   },
   "peerDependencies": {
     "karma": "*"
```

Comment:
In line with https://github.com/karma-runner/karma-jasmine/pull/304#issuecomment-1096059059, let's replace `*` with `^6.3.16`.

Reply:
I went with `^6.0.0` which makes more sense to me (it's a peer dependency and the idea is to allow compatible versions only), but we can change it.

Reply:
I have no experience on this. @devoto13 wdyt?

Reply:
`^6.0.0` sounds fine to me!